### PR TITLE
Resetting stack after successful transaction submission

### PR DESCRIPTION
**Pull request by RenanCostaNascimento**

### CODE REVIEW

File: src/scenes/SubmitTransaction/index.js [hunk 1]
```diff
@@ -77,7 +77,13 @@ class TransactionDetail extends Component {
...
     const { navigation } = this.props
     if (submitted) {
+      const resetAction = StackActions.reset({
+        index: 0,
+        actions: [NavigationActions.navigate({ routeName: 'App' })],
+        key: null
+      })
       const navigateToHome = NavigationActions.navigate({ routeName: 'Transactions' })
+      navigation.dispatch(resetAction)
       navigation.dispatch(navigateToHome)
```
Comment (RenanCostaNascimento) [inline src/scenes/SubmitTransaction/index.js]: Is this the best way to do this?

Comment (julianorafael) [reply src/scenes/SubmitTransaction/index.js]: `key: null` reseta o app todo. Aqui poderia ser só um reset normal na stack atual, que é a do balance, e depois navega pra transactions.

File: src/scenes/SubmitTransaction/index.js [hunk 2]
```diff
@@ -178,6 +180,18 @@ class TransactionDetail extends Component {
...
+  _getStackToReset = (transactionType) => {
+    if (transactionType === 'Transfer Asset' || transactionType === 'Transfer' ||
+      transactionType === 'Freeze' || transactionType === 'Unfreeze') {
+      return 'BalanceScene'
+    }
+    if (transactionType === 'Participate') {
+      return 'ParticipateHome'
+    }
+
+    return null
```
Comment (julianorafael) [inline src/scenes/SubmitTransaction/index.js]: what happens here?

Comment (RenanCostaNascimento) [reply src/scenes/SubmitTransaction/index.js]: If the return is null, the reset action won't be called at line 86.

Comment (julianorafael) [reply src/scenes/SubmitTransaction/index.js]: got ya, fair point. Good stuff. LGTM.

**Pull request merged**